prov/shm/src/smr_init.c

```diff
@@ -167,6 +167,8 @@ static int smr_getinfo(uint32_t version, const char *node, const char *service,
 		if (cur->caps & FI_HMEM) {
 			if (!(mr_mode & FI_MR_HMEM)) {
 				fi_freeinfo(cur);
+				FI_WARN(&smr_prov, FI_LOG_CORE,
+					"mr_mode does not match FI_HMEM capability.\n");
```

Comment:
I think this should be FI_INFO, rather than a warning.  @aingerson can confirm.

Reply:
Yeah, I think this should be FI_INFO

Reply:
In this file, all error logs are used FI_WARN. Should we keep it consistent or change them all to use FI_INFO?

Reply:
The warning here is more similar to an fi_getinfo ENODATA capability/mode mismatch which are logged under FI_INFO in the rest of the code (in the util_getinfo path that is called above). See for example where the utility code checks the MR modes - https://github.com/ofiwg/libfabric/blob/main/prov/util/src/util_attr.c#L554
The other warnings in this file can be kept as warnings since those are space/device issues.